README.md

```diff
@@ -1 +1,77 @@
-# Berry-IMU
\ No newline at end of file
+# Berry-IMU
```

Comment:
Let's add the meta json too in this pr.

Reply:
That gets generated when you upload the module right? will add